KeyDistribution_BB84/KeyDistribution_BB84.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <PlatformTarget>x64</PlatformTarget>
+    <IsPackable>false</IsPackable>
+    <RootNamespace>Quantum.Kata.KeyDistribution</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Quantum.Canon" Version="0.5.1904.1302" />
```

Comment:
The errors you mentioned come from these lines - the NuGet packages for the project are still on 0.5 version. These three lines should be 

    <PackageReference Include="Microsoft.Quantum.Standard" Version="0.9.1908.2906" />
    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.9.1908.2906" />
    <PackageReference Include="Microsoft.Quantum.Xunit" Version="0.9.1908.2906" />

This produces a new and interesting set of errors, but those are manageable :-)

Reply:
Ah, got it! Thank you! I will fix those lines and deal with the resulting errors :)